docs/userguide/transforms/general/index.md

```diff
@@ -51,8 +54,13 @@ transliteration is not translation. Rather, script transliteration it is the
 conversion of letters from one script to another without translating the
 underlying words. The following shows a sample of script transliteration:
 
-Source Transliteration キャンパス kyanpasu Αλφαβητικός Κατάλογος Alphabētikós
-Katálogos биологическом biologichyeskom * Some of the characters may not be
+| Source | Transliteration |
+|---|---|
+| キャンパス | kyanpasu |
+| Αλφαβητικός Κατάλογος | Alphabētikós Katálogos |
+| биологическом | biologichyeskom |
+
+:point_right: **Note**: *Some of the characters may not be
```

Comment:
ditto -- and same for all notes below apparently

Reply:
Did global search and replace